src/main/java/com/academy/ui/components/AddClubPopUpComponent/AddClubPopUpStepOne.java

```diff
@@ -70,12 +74,18 @@ public class AddClubPopUpStepOne extends AddClubPopUpContainer {
     @FindBy(xpath = "./descendant::span[@class='ant-select-selection-item']")
     private WebElement centerSelectedTitle;
 
-    @FindBy(xpath = "//div[@class='ant-select-item-option-content']")
+    @FindBy(xpath = "//div[@class='ant-select-item ant-select-item-option']")
     private List<WebElement> centersList;
 
     @FindBy(xpath = "./descendant::span[@class='ant-select-selection-placeholder']")
     private WebElement selectPlaceholder;
 
+    @FindBy(xpath = "./descendant::div[contains(@class, ' add-club-select')]")
+    private WebElement centerSelectedInEditClub;
```

Comment:
Duplicate xpath, we have centerSelect
It is better to change xpath for centerSelect 
"./descendant::input[contains(@id,'centerId')]"